sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -299,22 +299,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
 
     this._logger.verbose(`${this.logPrefix} permanently closing this link.`);
 
-    // Remove the underlying AMQP link from the cache
-    switch (this._linkType) {
```

Comment:
Can we get rid of linkType now or is it still used for other things?

Reply:
It's used when we're forming (and reforming on update) the log prefix. So not yet, but it's cosmetic now.